app/BUILD.bazel

```diff
@@ -642,145 +667,103 @@ account for altered package and file name.
  )
 for file in ALTERED_TESTS]
 
+# TODO(#973): Fix app module tests for Roboelectric.
```

Comment:
Can you actually add this at each test declaration for currently expected failing tests?

Reply:
Done